{
    "source_id": "vcdb",
    "timeline": {
        "incident": {
            "year": 2020,
            "month": 3
        },
        "discovery": {
            "unit": "Months",
            "value": 7
        }
    },
    "victim": {
        "government": [
            "NA"
        ],
        "country": [
            "US"
        ],
        "region": [
            "019021"
        ],
        "victim_id": "Vertafore",
        "employee_count": "101 to 1000",
        "industry": "511210",
        "state": "FL"
    },
    "action": {
        "error": {
            "variety": [
                "Publishing error"
            ],
            "vector": [
                "Carelessness"
            ]
        }
    },
    "actor": {
        "internal": {
            "variety": [
                "Unknown"
            ],
            "motive": [
                "NA"
            ]
        }
    },
    "asset": {
        "assets": [
            {
                "variety": "S - Web application"
            }
        ],
        "cloud": [
            "Unknown"
        ],
        "role": [
            "IT"
        ]
    },
    "attribute": {
        "confidentiality": {
            "data": [
                {
                    "amount": 27700000,
                    "variety": "Personal"
                }
            ],
            "data_victim": [
                "Customer"
            ],
            "state": [
                "Stored unencrypted"
            ],
            "data_disclosure": "Yes",
            "data_total": 27700000
        }
    },
    "discovery_method": {
        "unknown": true
    },
    "plus": {
        "event_chain": [
            {
                "action": "err",
                "actor": "int",
                "asset": "srv",
                "attribute": "cp",
                "summary": "Data stored on public facing system in error was access by unauthorized peoples."
            }
        ],
        "modified": "2021-11-18T22:54:58.812Z",
        "created": "2021-11-18T22:54:58.812Z",
        "master_id": "df2ea3a1-edcb-46da-b37d-ee2be4f73c68",
        "analysis_status": "Reviewed",
        "analyst": "swidup",
        "dbir_year": 2022,
        "github": "16595"
    },
    "schema_name": "vcdb",
    "incident_id": "8a5dafe0-4897-11ec-be64-495e033b0bce",
    "security_incident": "Confirmed",
    "summary": " Info of 27.7 million Texas drivers exposed in Vertafore data breach  Vertafore blames incident on human error after user data was stored on an unsecured external storage service. The files were accessed by an external party.",
    "reference": "https://www.zdnet.com/article/info-of-27-7-million-texas-drivers-exposed-in-vertafore-data-breach/; https://vertafore.kroll.com/; https://www.click2houston.com/news/local/2020/11/14/texas-drivers-license-information-possibly-exposed-during-data-breach-company-says/",
    "confidence": "High",
    "targeted": "NA",
    "schema_version": "1.3.6"
}